requirements/base.txt

```diff
@@ -18,7 +18,7 @@ porter2stemmer==1.0
 unicodecsv==0.14.1
 metafone==0.5
 le-utils==0.1.15
-kolibri_exercise_perseus_plugin==1.1.8
```

Comment:
This version does not appear to be able available on PyPy,

Reply:
correct - see comment above.

Waiting for https://github.com/learningequality/kolibri-exercise-perseus-plugin/pull/195

Reply:
if you have the ability to review/release that I'd really appreciate it!